AnimatedWordCloud/Animator/TimelapseWordVector.py

```diff
@@ -7,9 +7,10 @@
 Handful class of containing timelapse data of word vectors.
 """
 
+from __future__ import annotations
 from typing import Dict, Tuple, List
```

Comment:
You may not need this line since you are importing annotations.
See https://zenn.dev/yaakiyu_s/articles/0dfff7c3dd86b7

Reply:
Fixed

Reply:
solved